.github/support.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for Support Requests - https://github.com/dessant/support-requests
+
+# Label used to mark issues as support requests
+supportLabel: support
+
+# Comment to post on issues marked as support requests, `{issue-author}` is an
+# optional placeholder. Set to `false` to disable
+supportComment: |
+  Hi there @{issue-author} :wave:!
+
+  I closed this issue because it was labelled as a support question.
+
+  Please repost this on the http://discourse.jupyter.org/ forum (where most of the people who hang out here also hang out, as well as more people).
+  We are trying to streamline the different discussion places and want to use the issues on GitHub repos for technical discussions on how to change the contents of the repo.
+  More general discussions should go to Discourse so that they are easier to find, better indexed by google and generally more accessible than being hidden in the bowls of a GitHub repository :smiley:
```

Comment:
```suggestion
  More general discussions should go to Discourse so that they are easier to find, better indexed by google and generally more accessible than being hidden in the bowels of a GitHub repository :smiley:
```
Spotted a typo!